toggl/api/models.py

```diff
@@ -641,8 +649,8 @@ def all_from_reports(self, start=None, stop=None, workspace=None, config=None):
             if not returned.get('data'):
                 return
 
-            for entity in returned.get('data'):
-                yield self._deserialize_from_reports(config, entity)
+            for entity in returned.get("data"):
```

Comment:
```suggestion
            for entity in returned.get('data'):
```

Let us stay consistent with the project...

Reply:
Ah, sorry about that. Feel free to squash the commits before merging.